lib/util/github/graphql/query-adapters/releases-query-adapter.ts

```diff
@@ -30,7 +30,24 @@ const query = prepareQuery(`
   }
 `);
 
+const GithubGraphqlRelease = z.object({
+  version: z.string(),
+  releaseTimestamp: z.string(),
+  isDraft: z.boolean(),
+  isPrerelease: z.boolean(),
+  url: z.string(),
+  id: z.number(),
+  name: z.string().nullable(),
+  description: z.string().nullable(),
+});
+export type GithubGraphqlRelease = z.infer<typeof GithubGraphqlRelease>;
+
 function transform(item: GithubGraphqlRelease): GithubReleaseItem | null {
+  const releaseItem = GithubGraphqlRelease.safeParse(item);
+  if (!releaseItem.success) {
+    return null;
```

Comment:
Will this filter out any results which have a missing field?

Reply:
Yes, instead of pretending they're present

Reply:
Actually, I don't think we'll have any, but if we do, we'll know about it from logs

Reply:
But this could suddenly change user experience, like happened for Packagist? We might filter out releases and autoclose PRs because of a missing `isDraft` or `url` field, for example?

Reply:
I double-checked everything and fixed one field. Some fields such as `releaseTimestamp` and `target` are nullable according to GitHub GraphQL API docs, but we can't handle these nulls, so they'll be filtered.

Reply:
So if releaseTimestamp is missing then we'll filter out the release?

Reply:
Yes, because we can't tell whether it fresh or not. I believe, if it ever was the case, we'd catch runtime errors.